modules/statistics/RTPStatsCollector.js

```diff
@@ -84,6 +84,10 @@ SsrcStats.prototype.setCodec = function(codec) {
     this.codec = codec || '';
 };
 
+SsrcStats.prototype.setEncodeStats = function(encodeStats) {
+    this.encodeStats = encodeStats || {};
```

Comment:
Nope, they are not reset elsewhere. We'd be using this to show the codec for screenshare in the stats (currently not implemented but on my list) since we can have separate codecs for camera and screenshare.